.env

```
@@ -8,6 +8,8 @@ BOLTZ_BACKEND_IMAGE=boltz/boltz:latest
 BOLTZ_WEBAPP_IMAGE=boltz/web-app:latest
 BOLTZ_CLIENT_IMAGE=boltz/boltz-client:latest
 FOUNDRY_IMAGE=boltz/foundry:latest
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FOUNDRY_IMAGE key should go before the LND_IMAGE key

(UnorderedKey)